model/src/main/java/org/mskcc/cmo/metadb/model/SampleManifestEntity.java

```diff
@@ -22,12 +21,10 @@
     @Relationship(type = "PX_TO_SP", direction = Relationship.INCOMING)
     private PatientMetadata patient;
     @Relationship(type = "SAMPLE_MANIFEST", direction = Relationship.OUTGOING)
-    private SampleManifest sampleManifest;
-
-    public SampleManifestEntity() {
-        super();
-    }
+    private SampleManifestJsonEntity sampleManifestJsonEntity;
 
+    public SampleManifestEntity() {}
```

Comment:
can you add the `super()` back?

Reply:
the default constructor is still missing  `super()`, once this is back we can rebase and merge